prvocisla.cpp

```diff
@@ -29,16 +29,19 @@ int * sieve(int n){
 
     for (int p = 2; p<= n; p++){
         if (A[p] != 0) numPrime++;
+    
     }
 
     L = (int *) malloc(numPrime*sizeof(int));
 
     int j = 0;
+    
     while (j < numPrime) {
```

Comment:
nechapu proc jsou ty smycky dve.... myslim ze je to spatne, ale zkus to nejak vysvetlit prvne

Reply:
dve smycky kde?

Reply:
> dve smycky kde?

zanorene (logicky) ... tato a ta pod tim... dve smycky na jednu iteraci... na pruchod jednim 1d polem...

Reply:
Jednoduchy check- hod si tam print a rekni mi kolikrat ta smycka probehne.